recipe/meta.yaml

```
@@ -1,50 +1,54 @@
 {% set version = "17.0.6" %}
 {% set major_ver = version.split(".")[0] %}
 
+# as of LLVM 17, we expect an "rc" suffix for the release candidates
+{% set extra = "rc" if version.split(".")[-1] not in "01234567" else "" %}
+{% set extra = "git" if version.split(".")[-1] == "dev0" else extra %}
+
 package:
   name: llvm-package
   version: {{ version }}
 
 source:
-  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
-  sha256: 58a8818c60e6627064f312dbf46c02d9949956558340938b71cf731ad8bc0813
+  url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-{{ version.replace(".rc", "-rc") }}.tar.gz
+  sha256: 81494d32e6f12ea6f73d6d25424dbd2364646011bb8f7e345ca870750aa27de1
   patches:
+    - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
     - patches/0002-restore-macos-10.9-support.patch
+    # this patch adds the needed version numbers to 
+    - patches/0003-change-osx-version-for-libllvm-c.patch
     # These two patches remove expected test failures.
     - patches/0004-remove-unsupported-intrinsic-test.patch
     - patches/0005-remove-permission-based-unit-tests.patch
-    # This patches cmake code which isn't hit in the current configuration, however it's kept in case we want to build
-    # libllvm-c in the future
-    - patches/osx_ver.patch
     # A different approach to this fix: https://github.com/conda-forge/llvmdev-feedstock/commit/1da42e54b055d26bc58da77bd1f88ad460ac90f4
-    - patches/MSVC_DIA_SDK_path_fix.diff  # [win]
+    # - patches/MSVC_DIA_SDK_path_fix.diff  # [win]
 
 build:
-  number: 1
+  number: 3
   merge_build_host: false
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake
     - ninja
-    - python >=3
+    - python
     - libcxx {{ cxx_compiler_version }}  # [osx]
     - patch       # [not win]
     - m2-patch    # [win]
-    - git         # [(linux and x86_64)]
+    - git
```

```suggestion
    - git  # [not win]
```
Our CI already has `git` on `win`. Adding it there can cause troubles